docs/docs/workflows/change-measurements.md

```diff
@@ -0,0 +1,14 @@
+# Changing measurements
+
+There might be situations in which you have a running Constellation cluster, but want to change the expected measurements.
```

Comment:
I'm unsure if this should be its own workflow. Can we think of any use case besides fixing attestation failures? If yes, there should be a motivation at the beginning of the workflow. If not, there should either also be a motivation at beginning and in addition an entry on the Troubleshooting page linking to this, or the content should be completely moved to the troubleshooting page.

Reply:
Can't think of other reasons right now. Moved to troubleshooting.